Homework 4

LeahLeah88 opened the pull request:
Домашняя работа №4, Лосева Ольга.

LeahLeah88 committed:
Homework 4

tak333 commented on HW-4-V8-Fursova-AU/ConsoleApplication14.cpp:
```diff
@@ -0,0 +1,17 @@
+	int n;
+	std::cout << "n = ";
+	std::cin >> n;
+	std::cout << Calculate (n);
```
Здесь нужно реализовать с использованием цикла, без рекурсии.

tak333 reviewed (changes requested):
Переименуйте репозиторий + у вас очень много файлов загружается на гит.
Ну и замечание учтите.

LeahLeah88 committed:
Delete HW-4-V8-Fursova-AU/.vs directory

Удаление папки .vs

LeahLeah88 committed:
Delete HW-4-V8-Fursova-AU/x64/Debug directory

Удаление папки x64.

LeahLeah88 committed:
Delete HW-4-V8-Fursova-AU/ConsoleApplication14.user

Удаление файла  ConsoleApplication14.user.

LeahLeah88 committed:
Delete HW-4-V8-Fursova-AU/ConsoleApplication14.vcxproj.user

Удаление файла ConsoleApplication14.vcxproj.user

LeahLeah88 committed:
Delete HW-4-V8-Fursova-AU/HW-4-V8-Fursova-AU.user

Удаление файла HW-4-V8-Fursova-AU.user

LeahLeah88 committed:
Delete ConsoleApplication14.sln

Удаление файла ConsoleApplication14.sln

LeahLeah88 committed:
Delete HW-4-V8-Fursova-AU directory

Удаление папки HW-4-V8-Fursova-AU

LeahLeah88 committed:
Add project files.

	
Add project files.

LeahLeah88 committed:
Исправлено замечание, задача реализована с использованием цикла.

Исправлено замечание, задача реализована с использованием цикла.

LeahLeah88 committed:
Подключение файла с кодом.

Подключение файла с кодом.

LeahLeah88 committed:
Add project files.

Add project files.

tak333 commented on HW-4/HW-4.cpp:
```diff
@@ -0,0 +1,25 @@
+	std::cout << "n = ";
+	std::cin >> n;
+	int number = 3;			//номер искомого элемента
+	int element;			//искомый элемент
```
Объявляйте переменную в цикле, если она только там нужна.

LeahLeah88 replied:
Исправлено

tak333 reviewed (changes requested):
Учтите комментарий и переименуйте репозиторий: в имени указываются ваши ФИО.

LeahLeah88 committed:
Update HW-4.cpp

Переменная element теперь объявляется в цикле, так как она используется только в нём
